quickstarts/microsoft.datafactory/data-factory-v2-azure-sql-database-to-sql-data-warehouse-copy/prereqs/prereq.azuredeploy.parameters.json

```diff
@@ -6,7 +6,10 @@
             "value": "GEN-UNIQUE"
         },
         "databaseAdminPassword": {
-            "value": "GEN-PASSWORD"
+            "value": "GEN-SELFSIGNED-CERT-PASSWORD"
+        },
+        "sqlServerBase": {
```

Comment:
remove this param and use the environment function to get the endpoint

Reply:
Fixed